src/main/java/org/apache/sysds/runtime/controlprogram/paramserv/FederatedPSControlThread.java

```diff
@@ -360,7 +362,26 @@ protected void computeWithBatchUpdates() {
 			}
 		}
 	}
+	//****************************************  ATEFEH *********************************************************************
```

Comment:
we do not use author tags - so please remove such comments with your name.

Reply:
missing to remove it. well done and removed.